recommender/recommender.py

```diff
@@ -27,12 +27,37 @@ def emit(param1, param2):
 from mako.lookup import TemplateLookup
 
 from xblock.core import XBlock
-from xblock.fields import Scope, List, Dict
+from xblock.fields import Scope, List, Dict, Boolean, String
 from xblock.fragment import Fragment
 from xblock.reference.plugins import Filesystem
 
 from webob.response import Response
 
+
+def stem_url(url):
```

Comment:
%23 is the encoding for #. In both cases, the fragment is what comes after the #. If you go to the URL above, both Chrome and Firefox will convert the %23 to #. 

Regardless, the library swli found (Python urlparse+unquote_plus) have correct behavior.